commands/permissions.go

```diff
@@ -44,11 +53,40 @@ var ShowRoleCmd = &cobra.Command{
 	RunE:    withClient(showRoleCmdF),
 }
 
+var AssignUsersCmd = &cobra.Command{
+	Use:   "assign [role_name] [username...]",
+	Short: "Assign users to role (EE Only)",
+	Long:  "Assign users to a role by username (Only works in Enterprise Edition).",
+	Example: `  permissions assign system_admin john.doe jane.doe
+  permissions assign system_manager john.doe jane.doe
+  permissions assign system_user_manager john.doe jane.doe
+  permissions assign system_read_only_admin john.doe jane.doe`,
+	Args: cobra.MinimumNArgs(2),
+	RunE: withClient(assignUsersCmdF),
+}
+
+var UnassignUsersCmd = &cobra.Command{
+	Use:   "unassign [role_name] [username...]",
+	Short: "Unassign users from role (EE Only)",
+	Long:  "Unassign users from a role by username (Only works in Enterprise Edition).",
+	Example: `  permissions unassign system_admin john.doe jane.doe
+  permissions unassign system_manager john.doe jane.doe
+  permissions unassign system_user_manager john.doe jane.doe
+  permissions unassign system_read_only_admin john.doe jane.doe`,
+	Args: cobra.MinimumNArgs(2),
+	RunE: withClient(unassignUsersCmdF),
+}
+
 func init() {
+	AddPermissionsCmd.Flags().Bool(flagAncillaryPermissions, false, "Optional. Add all of the ancillary permissions used by each sysconsole_* permission being added.")
```

Comment:
We could make this a `BoolP` flag to support `-a` too

```suggestion
	AddPermissionsCmd.Flags().BoolP("ancillary", "a", false, "Optional. Add all of the ancillary permissions used by each sysconsole_* permission being added.")
```